elytron/src/main/java/org/wildfly/extension/elytron/AuditResourceDefinitions.java

```diff
@@ -87,6 +87,11 @@
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition FLUSHED = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.FLUSHED, ModelType.BOOLEAN, true)
```

Comment:
I'd suggest `autoflush` to be consistent with what the logging subsystem is using.